AODN/AODN-WAVE-NRT/ARDC_OMC_API_NRT/config/sources_metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+  "b7b3ded0-6758-4006-904f-db45f8cc012e": {
+    "_variables": {
+        "TIME": {
+            "long_name": "this is a test"
+        }
+    },
+    "id": "b7b3ded0-6758-4006-904f-db45f8cc012e",
+    "name": "B10",
+    "site_name": "",
+    "site_code": "",
+    "long_name": "Beacon 10",
+    "institution": "omc",
+    "deployment_start_date": "2021-11-18T07:12:36.345066Z",
+    "latitude_nominal": "",
+    "longitude_nominal": ""
```

Comment:
@bpasquer how do you currently do your location test for BOM buoys? Do you use lat_min and lat_max or do you use lat_nominal and a maximum distance in degrees east from lat_nominal? What do you think would be best?